pamelalozano/src/main/webapp/index.html

```diff
@@ -224,9 +224,12 @@ <h3>MongoDB</h3>
 
     <div id="comments-section">
       <p style="text-align: center;">
-        <i class="fa fa-chevron-circle-left fa-2x" aria-hidden="true" style="float: left;"></i>
+
+        <i class="fa fa-chevron-circle-left fa-2x" aria-hidden="true" style="float: left; cursor: pointer;" onclick="changePage('back')"></i>
```

Comment:
You could hide the "back" icon if the current page is 1.

Also hide the "next" icon if there are no more results. I think that would be a little harder to figure out.

Reply:
Sure I guess I can toggle the class according to the data response but I think I'll do it when I'm done with week 4 :) Thank you!